src/commands/cli/tarballs/smoke.ts

```diff
@@ -45,11 +46,7 @@ export default class SmokeTest extends SfCommand<void> {
 
   public async run(): Promise<void> {
     this.flags = (await this.parse(SmokeTest)).flags;
-    const executables = [path.join('tmp', this.flags.cli, 'bin', this.flags.cli)];
-    if (this.flags.cli === CLI.SFDX) {
-      executables.push(path.join('tmp', this.flags.cli, 'bin', CLI.SF));
-    }
-    await Promise.all(executables.map((executable) => this.smokeTest(executable)));
```

Comment:
Feels good to rip out `sfdx` logic 🎉

Reply:
The `cli` flag could be changed to a string with `sf` as the default.

Reply:
I just removed the flag altogether